test/functional/org_admin/templates_controller_test.rb

```diff
@@ -513,28 +513,32 @@ def verify_own_templates_table(user)
   end
   
   def verify_funder_templates_table(user)
-    assert_select "#funder-templates table tbody" do |el|
-      # An Org Admin should see all of the funder/default templates (except ones that belong to their org)
-      templates = Template.where("(org_id IN (?) OR is_default = ?) AND org_id != ?", Org.where(org_type: [2,3]).collect(&:id), true, user.org.id)
-      if user.can_org_admin?
-        templates.each do |template|
-          # Expect to only see published public templates
-          if template.publicly_visible? && template.published?
-            assert el.to_s.include?(template.title), "expected #{user.email}'s customizable table to have the funder (or default) template: '#{template.title}'" 
-          else
-            assert_not el.to_s.include?(template.title), "expected #{user.email}'s customizable table to NOT have the unpublished/non-public funder template: '#{template.title}' (from org: #{template.org.abbreviation})"
+    if user.org.funder_only?
+      assert_select "#funder-templates table tbody", 0, "expected a funder only Org to NOT see the customizable table"
```

Comment:
it is nice to see assert_select here to inspect the HTML resulting however I would place these tests as integration tests instead.

Reply:
yes. we have a bunch of these assert_selects around. Lets move after MVP when we have a chance to reevaluate the tests.  We need to do a lot of work there